src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -169,45 +199,47 @@ export default function Filters({ page }: FilterProps) {
                 />
               }
             </div>
-            <div>
+        
+            <div ref={unityAlignedOnlyItem}>
               {buildFilterCheckbox('unity_aligned_only', Translate('UnityStudiesOnly'), Translate('UnityStudiesOnlyLong'), "https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/early-investigations")}
             </div>
-            <div>
+            <div ref={sourceTypeItem}>
               {buildFilterDropdown('source_type', Translate('SourceType'))}
             </div>
-            <div>
+            <div ref={overallRiskOfBiasItem}>
               {buildFilterDropdown('overall_risk_of_bias', Translate('OverallRiskOfBias'))}
             </div>
           </div>
           <div className="pb-1">
             <div>
               <SectionHeader header_text={Translate('Demographics')} tooltip_text={Translate('DemographicsTooltip')}/>
             </div>
-            <div>
+            <div ref={populationGroupItem}>
               {buildFilterDropdown('population_group', Translate('PopulationGroup'))}
             </div>
-            <div>
+            <div ref={sexItem}>
               {buildFilterDropdown('sex', Translate('Sex'))}
             </div>
-            <div>
+            <div ref={ageItem}>
               {buildFilterDropdown('age', Translate('Age'))}
             </div>
           </div>
           <div className="pb-1">
             <div>
               <SectionHeader header_text={Translate('TestInformation')} tooltip_text={Translate('TestInformationTooltip')}/>
             </div>
-            <div>
+            <div ref={testTypeItem}>
               {buildFilterDropdown('test_type', Translate('TestType'))}
             </div>
-            <div>
+            <div ref={isotypesReportedItem}>
               {buildFilterDropdown('isotypes_reported', Translate('IsotypesReported'))}
             </div>
-            <div>
+            <div ref={antibodyTargetItem}>
               {buildFilterDropdown('antibody_target', Translate('AntibodyTarget'))}
             </div>
           </div>
         </div>
+        <Button color="grey" size="small" onClick={clearFilter}> {Translate('ClearAllFilters')}</Button>

```

Comment:
Let's make this button disabled unless there are filters to clear. I think that would make for a better user experience

Reply:
Done!